crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -1209,18 +1200,12 @@ mod tests {
         formatted_body: &str,
         ts: u64,
     ) -> TimelineEvent {
-        TimelineEvent::new(sync_timeline_event!({
-            "event_id": "$eventid6",
-            "sender": user_id,
-            "origin_server_ts": ts,
-            "type": "m.room.message",
-            "room_id": room_id.to_string(),
-            "content": {
-                "body": body,
-                "format": "org.matrix.custom.html",
-                "formatted_body": formatted_body,
-                "msgtype": "m.text"
-            },
-        }))
+        EventFactory::new()
```

Comment:
If you're interested in bonus points :D This method `message_event` is used only in two callers, so we could actually inline it there, and reuse the same `EventFactory` in that case. I think I'd like it slightly better, because then we see immediately what are the events, as code readers, and we don't have to do an "context switch" in the reading. Of course, that's totally optional and we could do it later if you don't feel like it :)

Reply:
Done in 14f603a